apps/sertra/styles/globals.css

```diff
@@ -1,10 +1,25 @@
+:root {
+  --color-babyblue: #ccddfe;
+  --color-blue: #242e48;
+  --color-coolgray: #93a0c2;
+  --color-darkblue: #17213b;
+  --color-highlightgreen: #02ff95;
+  --color-highlightred: #ff3a52;
+  --color-lightblue: #353e58;
+  --color-lightblue115: #49567a; /* Lightened by 15% */
+  --color-orange: #ff956b;
+  --color-orange99: #fc9347; /* Darkened by 1% */
+
+  --font-family-body: Exo2, sans-serif;
+  --font-family-header: Raleway, serif;
```

Comment:
Raleway is a sans-serif typeface, so I suggest to edit the fallback accordingly.

Reply:
Fixed